aws/src/test/java/org/apache/iceberg/aws/s3/TestS3FileIO.java

```diff
@@ -81,7 +81,9 @@ public class TestS3FileIO {
           "s3.write.tags.tagKey1",
           "TagValue1",
           "s3.delete.batch-size",
-          Integer.toString(batchDeletionSize));
+          Integer.toString(batchDeletionSize),
+          "s3.delete.num-threads",
+          "1");
```

Comment:
S3Mock is not handling concurrent deletions well and now that multi-threading batch delete is passing, the testPrefixDelete is failing.. Our integ tests should give us good coverage in this regard.